Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:alpine
 RUN apk add --update make gcc python3-dev musl-dev libffi-dev openssl openssl-dev
 WORKDIR /app
 RUN pip install pipenv
-COPY Pipfile Pipfile.lock /app/
-RUN pipenv install --deploy --system
+COPY Pipfile /app/
+RUN pipenv install --system --skip-lock
```

Comment:
This means we're installing the latest versions of all dependencies whenever we build the docker image. Unless we pin all dependencies in the Pipfile, then I don't think this is a good idea.

Reply:
Great point; originally, I wasn't intending on doing so, but it just hadn't crossed my mind.  @chartjes?